pkgs/development/misc/juce/default.nix

```diff
@@ -43,28 +57,47 @@ stdenv.mkDerivation (finalAttrs: {
     makeWrapper
   ];
 
-  buildInputs = [
-    freetype # libfreetype.so
-    curl # libcurl.so
-    (lib.getLib stdenv.cc.cc) # libstdc++.so libgcc_s.so
-    pcre # libpcre2.pc
-  ] ++ lib.optionals stdenv.hostPlatform.isLinux [
-    alsa-lib # libasound.so
-    libglvnd # libGL.so
-    webkitgtk_4_0 # webkit2gtk-4.0
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    darwin.apple_sdk.frameworks.Cocoa
-    darwin.apple_sdk.frameworks.MetalKit
-    darwin.apple_sdk.frameworks.WebKit
-  ];
+  buildInputs =
+    [
+      freetype # libfreetype.so
+      curl # libcurl.so
+      (lib.getLib stdenv.cc.cc) # libstdc++.so libgcc_s.so
+      pcre2 # libpcre2.pc
+      libsysprof-capture
+      libthai
+      libdatrie
+      lerc
+      libepoxy
+      sqlite
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      alsa-lib # libasound.so
+      libglvnd # libGL.so
+      webkitgtk_4_0 # webkit2gtk-4.0
+      util-linuxMinimal
+      libselinux
+      libsepol
+      libXdmcp
+      libxkbcommon
+      libXtst
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.apple_sdk.frameworks.Cocoa
+      darwin.apple_sdk.frameworks.MetalKit
+      darwin.apple_sdk.frameworks.WebKit
+    ];
+
+  propagatedBuildInputs = [ fontconfig ];
```

Comment:
Why is that propagated? Should we not use a wrapper instead?

Reply:
Because `ysfx`, and possibly other non-packaged software, expect `fontconfig` to be available when they require `juce`.

Reply:
@SuperSandro2000 Are you satisfied with the explanation?